src/lib.rs

```diff
@@ -21,7 +21,7 @@ use proofs::ProofState;
 use symbolic_expressions::Sexp;
 
 use ast::*;
-use typechecking::{TypeInfo, UNIT_SYM};
+pub use typechecking::{TypeInfo, UNIT_SYM};
```

Comment:
Must make public so that can access `TypeInfo` which is an argument to a method on sorts